app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= _("Foreman audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= n_('Displaying %{num_audits} of %{total_audits} audits',
+         'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
+  <% @audits.each do |audit| %>
+    <%= audit.username %>
+    ( <%= audit.remote_address %> )
+    <%= audit_action_name audit%> <%= audited_type audit %>:
+    <%= audit_title(audit) %>
+    -
```

Comment:
Unsure if this is needed if they're all on one line?  Or a blank line might be clearer, e.g.

<pre>
»Foreman audit summary«
=================================
»Displaying  of 1 audits«

Admin User (127.0.0.1) updated Host: cobalt.redhat.com
http://foreman.example.com/audit/123

Admin User (127.0.0.1) updated Host: cobalt.redhat.com
http://foreman.example.com/audit/124
</pre>